tailwind.config.js

```diff
@@ -77,12 +77,13 @@ module.exports = {
       demo: {
         white: 'var(--demo-white)',
         black: 'var(--demo-black)',
-        darkBlue: 'var(--demo-darkBlue)',
-        mediumBlue: 'var(--demo-mediumBlue)',
-        lightBlue: 'var(--demo-lightBlue)',
-        darkGray: 'var(--demo-darkGray)',
-        mediumGray: 'var(--demo-mediumGray)',
-        lightGray: 'var(--demo-lightGray)',
+        softBlack: 'var(--demo-soft-black)',
+        darkBlue: 'var(--demo-dark-blue)',
+        mediumBlue: 'var(--demo-medium-blue)',
+        lightBlue: 'var(--demo-light-blue)',
+        darkGray: 'var(--demo-dark-gray)',
+        mediumGray: 'var(--demo-medium-gray)',
+        lightGray: 'var(--demo-light-gray)',
```

Comment:
The variable names here will need to be in the format of:
* `soft-black`
* `dark-blue`
* etc.

This will ensure the tailwind utility classes are used as `bg-demo-soft-black`. 

This is the change that will require a search are replace through any instances of `*.stories` files for the tailwind variable change.

Reply:
Tired this and it blew everything up. Forgot to wrap the keys in quotes. Fixed.